resources/i18n/ko_KR/uranium.po

```diff
@@ -136,13 +135,13 @@ msgstr "파일이 이미 있음"
 #, python-brace-format
 msgctxt "@label Don't translate the XML tag <filename>!"
 msgid "The file <filename>{0}</filename> already exists. Are you sure you want to overwrite it?"
-msgstr "파일이 이미 있습니다. 덮어 쓰시겠습니까?"
+msgstr "<filename>{0}</filename> 파일이 이미 있습니다. 덮어 쓰시겠습니까?"
```

Comment:
Thank you! We had seen that these were missing the `{0}` and such but we didn't have time to tell our translators this and make another translation round before the release. But you fixed it! That's very nice!

Reply:
Ok, I'll try it to 3.3 branch.
Anyway, do you have Korean translator? Wow.

Reply:
Yeah we hire a translation bureau (Bothof) that does most of our translations.